src/quad_expr.jl

```diff
@@ -269,28 +269,28 @@ last term added.
 function _fill_vqf!(terms::Vector{<:MOI.VectorQuadraticTerm}, offset::Int,
                     oi::Int, aff::AbstractJuMPScalar)
     i = 1
-    for term in quadterms(aff)
+    for term in quad_terms(aff)
         terms[offset + i] = MOI.VectorQuadraticTerm(Int64(oi),
                                                     moi_quadratic_term(term))
         i += 1
     end
-    return offset + length(quadterms(aff))
+    return offset + length(quad_terms(aff))
 end
 
 function MOI.VectorQuadraticFunction(quads::Vector{QuadExpr})
-    num_quad_terms = sum(quad -> length(quadterms(quad)), quads)
-    quad_terms = Vector{MOI.VectorQuadraticTerm{Float64}}(undef, num_quad_terms)
-    num_aff_terms = sum(quad -> length(linear_terms(quad)), quads)
-    lin_terms = Vector{MOI.VectorAffineTerm{Float64}}(undef, num_aff_terms)
+    num_qua_terms = sum(quad -> length(quad_terms(quad)), quads)
+    qua_terms = Vector{MOI.VectorQuadraticTerm{Float64}}(undef, num_qua_terms)
```

Comment:
Not sure I'm a fan of `qua_terms`. The fact that `quad_terms` conflicts doesn't make it any better to use `qua`. What about `quadratic_terms`?